model.js

```diff
@@ -0,0 +1,2 @@
+const db = require('./db/connection');
+const format = require('pg-format');
```

Comment:
Please only add files when they relate to the current ticket (eg. you don't need a model file yet)